docs/index.rst

```diff
@@ -14,20 +14,26 @@ Notation and basic formulas are borrowed from [DermerMenon2009]_ which constitut
 
 Implementation
 ..............
-The numerical operations are delegated to `numpy arrays <https://numpy.org>`_, all the physical quantities computed are casted as `astropy Quantities <https://docs.astropy.org/en/stable/units/>`_. 
+The numerical operations are delegated to `numpy arrays <https://numpy.org>`_, all the physical quantities computed are casted as `astropy Quantities <https://docs.astropy.org/en/stable/units/>`_.
 
 License
 -------
 The code is licensed under `GNU General Public License v3.0 <https://www.gnu.org/licenses/gpl-3.0.html>`_ (see `LICENSE.md` in the main directory).
 
 Installation
 ------------
-The code is available in the `python package index <https://pypi.org/project/agnpy/>`_ and can be installed via `pip
+The code is available in the `python package index <https://pypi.org/project/agnpy/>`_ and can be installed via ``pip``
 
 .. code-block:: bash
 
     pip install agnpy
 
+The code can also be installed with ``conda``
+
+.. code-block:: bash
+
+    conda install -c conda-forge agnpy
```

Comment:
Again, if you prefer, the channel can be changed to `astropy`.

Reply:
It's fine as is